app/services/document_capture_session_result.rb

```diff
@@ -35,6 +35,12 @@ def success_status
   alias_method :attention_with_barcode?, :attention_with_barcode
   alias_method :pii_from_doc, :pii
 
+  def self.deprecator
+    ActiveSupport::Deprecation.new('2024Q1', 'TrueIdRequest')
+  end
+
+  deprecate pii_from_doc: :dummy, deprecator: deprecator
```

Comment:
This is a new thing in the codebase, was it meant to be left in or is this for testing purposes?

Reply:
@charleyf , it's for testing.

Reply:
I remember we talked about deprecator in Timnit eng huddle.  It's good to see it in use.